build.sc

```diff
@@ -164,6 +186,10 @@ object implicits extends Module {
   class JvmModule(val crossScalaVersion: String) extends ImplicitsModule with CommonJvmModule{
     def moduleDeps = Seq(core.jvm())
     def artifactName = shade("weepickle-implicits")
+
+    override def mimaBinaryIssueFilters = Seq[ProblemFilter](
+      ProblemFilters.exclude[MissingClassProblem]("com.rallyhealth.weepickle.v1.implicits.internal.Macros*")
```

Comment:
I removed this file which was deprecated in #24. There have been two PRs that submit changes here instead of the right place.

It's only used at compile time, so removing it should not cause runtime dep hell.